R/utils-render.R

```diff
@@ -27,16 +45,11 @@ cleanTempSpace <- function() {
 #' @return the html file and other report bits
 #' 
 startRender <- function(data, author, reportName){
-  data <- data 
-  
-  wd <- getwd()
-  tmp_folder_name <- paste0('tmp-', reportName)
+  warning("This rendering style is deprecated, convert to using whisker style and startTemplatedRender")
   
-  output_dir <- paste0(wd, "/", tmp_folder_name)
+  data <- data 
```

Comment:
I think it puts data in the env so it's available to rmarkdown. I never quite wrapped my head around why but this is going away soon.